payara-micro/src/main/java/fish/payara/arquillian/container/payara/managed/PayaraMicroDeployableContainer.java

```diff
@@ -131,7 +131,7 @@ public void setup(PayaraMicroContainerConfiguration configuration) {
 
     @Override
     public ProtocolDescription getDefaultProtocol() {
-        return new ProtocolDescription("Servlet 4.0");
+        return new ProtocolDescription("Servlet 3.0");
```

Comment:
Any specific details on version change to `3.0`?

Reply:
I think it was changed by mistake in first place -- there's no Arquillian protocol implementation for `4.0`